scalafmt-tests/src/test/resources/scala3/OptionalBraces.stat

```diff
@@ -3598,3 +3598,35 @@ object a {
   val a = test("" +: Seq("").map { _ + "" }: _*)
   val a = test(("" +: Seq("").map { _ + "" })*)
 }
+<<< coloneol in fewer braces
+maxColumn = 80
+===
+object a:
```

Comment:
Could you also maybe add two more tests such as:
```scala
object a:
  def f(): Unit =
    List(1, 2, 3).foo: a =>
      a + 2
    .apply:
      12 + 2
```

and maybe:
```scala
object a:
  def f(): Unit =
    List(1, 2, 3).foo:
       case a : Int =>
       case _ =>
   otherTerm()
      
```

Reply:
thanks for the suggestion! also handled `.match<outdent>`.